pyClient/testERCTokenMixing.py

```diff
@@ -223,8 +227,9 @@ def mint_token(token_instance, spender_address, deployer_address, token_amount):
     )
 
     # Charlie tries to decrypt the ciphertexts from Bob's previous transaction
-    recovered_plaintext1 = zethUtils.receive(ciphertext_bob_to_charlie1, keystore["Charlie"]["AddrSk"]["dk"], "charlie")
-    recovered_plaintext2 = zethUtils.receive(ciphertext_bob_to_charlie2, keystore["Charlie"]["AddrSk"]["dk"], "charlie")
+    charlie_sk = zethUtils.get_private_key_from_bytes(keystore["Charlie"]["AddrSk"]["encSK"])
+    recovered_plaintext1 = zethUtils.receive(ciphertext_bob_to_charlie1, pk_sender_bob_to_charlie, charlie_sk, "charlie")
```

Comment:
You pass an instance of `PrivateKey` as input and a a public key in byte format.
Let's pass both a `PrivateKey` and a `PublicKey` as input to receive.

In other words:
```
charlie_sk = zethUtils.get_private_key_from_bytes(keystore["Charlie"]["AddrSk"]["encSK"])
recovered_plaintext1 = zethUtils.receive(ciphertext_bob_to_charlie1, pk_sender_bob_to_charlie, charlie_sk, "charlie")
```
Needs to become:
```
charlie_sk = zethUtils.get_private_key_from_bytes(keystore["Charlie"]["AddrSk"]["encSK"])
pk_sender_bob_to_charlie = zethUtils.get_public_key_from_bytes(pk_sender_bob_to_charlie)
recovered_plaintext1 = zethUtils.receive(ciphertext_bob_to_charlie1, pk_sender_bob_to_charlie, charlie_sk, "charlie")
```

And the `zethUtils.receive` and `zethUtils.(decrypt|encrypt)` functions need to be modified accordingly to have consistent types.

Reply:
Signatures modified. Now `encrypt/decrypt/receive` take in input sk and pk objects.